src/components/card.tsx

```diff
@@ -20,7 +20,7 @@ export const Card = forwardRef<
     id={id}
     className={clsx(
       'rounded-xl min-w-52 border border-grey-200',
-      theme === 'grey' && 'bg-[#efefef]',
+      theme === 'grey' && 'bg-grey-100',
```

Comment:
`#efefef` is an out-of-design-system color that was in the incentives designs mistakenly. This change will affect the gray cards in the incentives embed, but that's intentional.